eng/.docsettings.yml

```diff
@@ -63,6 +62,7 @@ known_content_issues:
   - ['sdk/appconfiguration/Azure.ApplicationModel.Configuration/README.md','#5499']
   - ['sdk/appconfiguration/Azure.ApplicationModel.Configuration/tests/Readme.md','#5499']
   - ['sdk/core/Azure.Core/README.md','#5499']
+  - ['sdk/storage/README.md','azure-sdk-tools/issues/42']
```

Comment:
nit:  Just a double-check....  are there are still content "errors" after your changes?

Reply:
Yes - this is the top level README which links to the different libraries.  It doesn't have its own Examples, etc.